Makefile

```diff
@@ -0,0 +1,16 @@
+##
+##usage :
+##-------
```

Comment:
Could you remove this comment?

Reply:
see https://github.com/EvanPurkhiser/tune-manager/pull/13#discussion_r520578195